orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/TerminateInstancesTask.groovy

```diff
@@ -45,8 +47,14 @@ class TerminateInstancesTask extends AbstractCloudProviderAwareTask implements T
         "terminate.account.name": account,
         "terminate.region"      : stage.context.region,
         "kato.last.task.id"     : taskId,
-        "kato.task.id"          : taskId, // TODO retire this.
-        "terminate.instance.ids": stage.context.instanceIds,
+        "terminate.instances"   : stage.context.instances,
     ])
   }
+
+  static Map convert(Stage stage) {
+    def operation = [:] << stage.context
+    def instances = operation.remove("instances")
+    operation.instanceIds = instances.collect { it.id }
```

Comment:
Are we restricted to terminating instances within a single zone in a given operation? Seems like it based on https://github.com/spinnaker/clouddriver/blob/master/clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/TerminateGoogleInstancesAtomicOperation.groovy#L71

Only asking because I'm trying to get multi-select operations into the UI for instances right now and thinking I need to batch them up by zone...unless you want to change the Clouddriver API? Eh? Don't you want to change the Clouddriver API, too? Eh?